Month2/week1/Month2-Project Week 1-Remembering the Bases/VideoGame/Steam.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "CategoryContainer.h"
+class FSteam 
+{
+private:
+	FCategoryContainer CategoryContainer;
+	FCategory Uncategorized;
+	enum class EMenuCommand {MainMenu, AddGames, Categories ,DisplayGames, Exit};
+	EMenuCommand ActiveCommand;
+	/*enum class Action { CreateCategory = 1, DeleteCategory };
+	Action ActionChoice;*/
+public:	
+	FSteam();
+	FSteam(FSteam& oldSteam);
```

Comment:
Keep in mind that copy constructors should use const references, not just references. You don't want to modify the other object